README.md

```diff
@@ -212,6 +212,12 @@ Report it to the package repository.
 You can't. Package registrations are **permanent**. A version can not be overwritten in the
 registry, and code cannot be deleted.
 
+#### Can my package be registered without an [OSI approved license](https://opensource.org/licenses)?
+
+No, sorry. The registry is maintained by volunteers, and we don't have a legal team that can thoroughly review licenses. It is very easy accidentally wander into legally murky territory when combining common OSI licenses[^1] like GPL with non-OSI licenses and we don't want to subject Julia users to that risk when installing packages registered in General. See [these](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-796671872) [comments](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-804196208) for more discussion.
```

Comment:
easy accidentally -> easy to accidentally

Reply:
Thanks!

```suggestion
No, sorry. The registry is maintained by volunteers, and we don't have a legal team that can thoroughly review licenses. It is very easy to accidentally wander into legally murky territory when combining common OSI licenses[^1] like GPL with non-OSI licenses and we don't want to subject Julia users to that risk when installing packages registered in General. See [these](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-796671872) [comments](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-804196208) for more discussion.
```